Makefile

```diff
@@ -44,9 +46,10 @@ push-ad-server: build-ad-server
 	$(PUSH_CMD) $(AD_SERVER_REPO_NAME)
 .PHONY: push-ad-server
 
-build-client:
+build-client: $(BUILDFILE_CLIENT)
+$(BUILDFILE_CLIENT): Makefile $(CLIENT_SRC_FILE)
 	$(BUILD_CMD) --tag $(CLIENT_NAME) --tag $(CLIENT_REPO_NAME) -f $(CLIENT_SRC_FILE) $(CLIENT_DIR)
-.PHONY: build-client
+	date -R >> $(BUILDFILE_CLIENT)
```

Comment:
Why date? Why not put the hex container id from the build command?
(Also the output of date -R is not machine friendly IMO)

Reply:
good point. updated to write container id and not date. (not sure why the machine friendly format for the content would matter. also i missed the point with the container id first and just wanted to put *something* in - now I only put the container id).